From a767610868167f29fc2cb2ddc32551ae5136c9f7 Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Wed, 4 Mar 2015 14:13:15 +0200
Subject: [PATCH 03/10] imx_v7_defconfig: enable Wilink8 related switches

* Disable internal mac80211 and cfg80211 as we are building wilink8
  drivers using backport and loading as modules
* enable wlan and hci related switches

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 arch/arm/configs/imx_v7_defconfig |   33 ++++++++++++++++++++++++++++++---
 1 file changed, 30 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 303e679..fac8d07 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -81,8 +81,10 @@ CONFIG_BT_HCIUART_BCSP=y
 CONFIG_BT_HCIUART_ATH3K=y
 CONFIG_BT_HCIBCM203X=y
 CONFIG_BT_ATH3K=y
-CONFIG_CFG80211=y
-CONFIG_MAC80211=y
+CONFIG_BT_HCIUART_LL=y
+CONFIG_BT_WILINK=y
+# CONFIG_CFG80211 is not set
+# CONFIG_MAC80211 is not set
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 # CONFIG_STANDALONE is not set
@@ -406,8 +408,33 @@ CONFIG_CRYPTO_DEV_FSL_CAAM_SM_TEST=y
 CONFIG_CRYPTO_DEV_FSL_CAAM_SECVIO=y
 CONFIG_CRC_CCITT=m
 CONFIG_CRC_T10DIF=y
-CONFIG_CRC7=m
+CONFIG_CRC7=y
 CONFIG_LIBCRC32C=m
 CONFIG_FONTS=y
 CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
+#TI add for WL8
+CONFIG_WLAN=y
+CONFIG_CRYPTO_ARC4=y
+CONFIG_CRYPTO_AES=y
+CONFIG_NETFILTER=y
+CONFIG_NETFILTER_ADVANCED=y
+CONFIG_NF_CONNTRACK=y
+CONFIG_NETFILTER_XTABLES=y
+CONFIG_NF_DEFRAG_IPV4=y
+CONFIG_NF_CONNTRACK_IPV4=y
+CONFIG_NF_CONNTRACK_PROC_COMPAT=y
+CONFIG_IP_NF_IPTABLES=y
+CONFIG_IP_NF_FILTER=y
+CONFIG_IP_NF_TARGET_LOG=y
+CONFIG_NF_NAT=y
+CONFIG_NF_NAT_NEEDED=y
+CONFIG_IP_NF_TARGET_MASQUERADE=y
+CONFIG_INPUT_UINPUT=y
+CONFIG_AVERAGE=y
+CONFIG_PROC_DEVICETREE=y
+# enable dynamic debug for wl8
+CONFIG_DYNAMIC_DEBUG=y
+#enable ti shared transport support support
+CONFIG_TI_ST=y
+CONFIG_ST_HCI=y
-- 
1.7.9.5

